### PR TITLE
Intro segue, section 2, section 4

### PR DESCRIPTION
**Pull request by lauterbur**
Three separate commits in this PR, sorry folks.

1. (edits to introduction, add segue paragraph) - minor edits to the introduction, plus added a segue paragraph to section 2.
2. (add section 2) - as it says, section 2 just lays out what this paper is/isn't
3. (add section 4 - intro to application/discussion) - as it says, section 4 lays out the hackathon stuff and how we're using it to illustrate what we've talked about re: necessary resources; still needs some numbers for stuff that's considered "in" the catalog

### CODE REVIEW

File: adding_species_manuscript.md [hunk 1]
```diff
@@ -135,7 +117,7 @@ since most population-level summaries of genetic variation do not take into acco
 (However, this may change in the future.)
 The differences between one whole-chromosome simulation and many independent simulations of smaller pieces of genome
 are due to linkage, i.e., correlations in transmission and inheritance between adjacent parts of the genome.
-In any simulation, even neutral ones,
+In any simulation, even neutral ones,c
```
Comment (RyanGutenkunst) [inline adding_species_manuscript.md]: Not a fan of this part of the commit. :-)

**Review by RyanGutenkunst — COMMENTED** [review]
LGTM, except for the one type added. (See inline comment.)

File: adding_species_manuscript.md (hunk 1 above)
Comment (lauterbur) [reply adding_species_manuscript.md]: Fair enough!

**Comment by lauterbur** [comment]
Rather than open a new PR, I added a commit here with the intro to the tutorial. I will merge the whole PR with any suggested changes shortly (tomorrow?).

**Pull request merged**